cocos/3d/CCMesh.cpp

```diff
@@ -125,13 +125,12 @@ Mesh::Mesh()
 : _skin(nullptr)
 , _visible(true)
 , _isTransparent(false)
+, _force2DQueue(false)
 , _meshIndexData(nullptr)
-, _material(nullptr)
 , _glProgramState(nullptr)
 , _blend(BlendFunc::ALPHA_NON_PREMULTIPLIED)
-, _visibleChanged(nullptr)
```

Comment:
`_visibleChanged` is an `std::function` and initializing an `std::function` with `nullptr` is equivalent to using its default constructor. Thus there is no need to keep this line.